lib/bridge.js

```diff
@@ -0,0 +1,13 @@
+const { ethers } = require("hardhat");
+
+async function deployBridge(regenBridge, toucanRegistry) {
+    const Bridge = await ethers.getContractFactory("ToucanRegenBridge");
+    const bridge = await Bridge.deploy(regenBridge, toucanRegistry);
```

Comment:
Has it been discussed:
- whether the `ToucanRegenBridge` is going to be upgradable
- who is going to be owner of `ToucanRegenBridge`
- whether `ToucanRegenBridge` needs to be ownable

Reply:
These are great questions to discuss as part of the design document but out of scope for this PR.